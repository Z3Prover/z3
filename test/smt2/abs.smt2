
; Copyright (c) 2015 Microsoft Corporation

(declare-const a Int)
(declare-const b Int)

(simplify (abs a))

(push)
(assert (< (abs a) 0))
(check-sat)
