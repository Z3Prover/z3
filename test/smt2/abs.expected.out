(ite (>= a 0) a (* (- 1) a))
unsat
